src/k8sconfiguration/azext_k8sconfiguration/tests/latest/test_kubernetesconfiguration_scenario.py

```diff
@@ -0,0 +1,73 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, record_only)
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class K8sconfigurationScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='cli_test_k8sconfiguration')
+    @record_only()
```

Comment:
why mark a record_only

Reply:
SourceControlConfiguration is an extension resource that requires a K8s cluster to be pre-created and certain actions be performed in the cluster directly (that are not ARM actions).  Since this is not possible during test run, per discussion with Xiaojian, marked the tests as record_only so they are only in replay mode and not part of the live runs.